samples/getting-started/cmake_caches.cmake

```diff
@@ -0,0 +1,63 @@
+# Copied from NDK's android.toolchain.cmake:
```

Comment:
i think this file could exist in `swift-android-toolchain` @ephemer

Reply:
Yeah it probably should